lib/internal/abort_controller.js

```diff
@@ -274,12 +276,25 @@ ObjectDefineProperty(AbortSignal.prototype, SymbolToStringTag, {
 
 defineEventHandler(AbortSignal.prototype, 'abort');
 
-function createAbortSignal(aborted = false, reason = undefined) {
+/**
+ * @param {{
+ *   aborted? : boolean,
+ *   reason? : any,
+ *   transferable? : boolean
+ * }} [init]
+ * @returns {AbortSignal}
+ */
+function createAbortSignal(init) {
```

Comment:
```suggestion
function createAbortSignal(init = kEmptyObject) {
```
and then requiring `kEmptyObject` from `internal/util` will probably fix the test failures. You might also want to rebase to include the change from https://github.com/nodejs/node/pull/43159 for this to work.